index.js

```diff
@@ -9,13 +9,14 @@ module.exports = {
     //default theme name is 'default'
     var options = app.options['ember-prism'] || { components: [] };
 
-    //import main stylesheet
-    app.import(app.bowerDirectory + '/prism/themes/prism.css');
-
     //import theme based on options
     if (options.theme){
-      app.import(app.bowerDirectory + '/prism/themes/prism-' + options.theme + '.css');
+      // allow ability to specify no css if we want to provide our own
+      if (options.theme !== 'none'){
+        app.import(app.bowerDirectory + '/prism/themes/prism-' + options.theme + '.css');
+      } 
     } else {
+      // fall back to default theme
       app.import(app.bowerDirectory + '/prism/themes/prism.css');
```

Comment:
Currently, without this PR the base theme effectively gets included twice if you don't specify a theme. [L13](https://github.com/brandonjmckay/ember-prism/blob/master/index.js#L13) & [L19](https://github.com/brandonjmckay/ember-prism/blob/master/index.js#L19). So the proposed flow would be: no theme? add default, theme? add specified, or none at all.